frontend/apps/erd-web/app/erd/p/[...slug]/page.tsx

```diff
@@ -65,11 +65,14 @@ export async function generateMetadata({
   const metaDescription =
     'Generate ER diagrams effortlessly by entering a schema file URL. Ideal for visualizing, reviewing, and documenting database structures.'
 
+  const imageUrl = '/assets/liam_erd.png'
+
   return {
     title: metaTitle,
     description: metaDescription,
     openGraph: {
       url: `https://liambx.com/erd/p/${joinedPath}`,
+      images: imageUrl ? [{ url: imageUrl }] : [],
```

Comment:
Q: Is there any reason to make it a ternary operator? imageUrl should always have a value.

Reply:
Oh, sorry!🙏 I was wrong. 
Fixed: https://github.com/liam-hq/liam/pull/514/commits/e5d9abf1a59b1639d12862b3853ba833d497a8b8
preview: https://liam-erd-gfbfdn7s1-route-06-core.vercel.app/erd/p/github.com/mastodon/mastodon/blob/main/db/schema.rb

Reply:
Since the imageUrl never becomes a falsy value, what about the following?

```ts
      images: [imageUrl],
```

Reply:
I understand🙏. I've removed the conditional check that wasn't needed!
https://github.com/liam-hq/liam/pull/514/commits/d1428d0b6cd044e43d2afb8dc8f5098fac3cae1a